keys/generate-key-pair.sh

```diff
@@ -31,6 +32,14 @@ if [[ -n $keyType ]] && [[ $keyType != "ecdsa256" ]] && [[ $keyType != "ecdsa384
   exit 2
 fi
 
+if [[ ${str} =~ OpenSSL ]]; then
+  echo "Found Homebrew Openssl"
+else
+	echo "You are not using the Homebrew version of OpenSSL. In order to run this script you must be using the Homebrew version of OpenSSL.
+	Go to this website and follow the instructions to set up your OpenSSL environment:
+	https://medium.com/@maxim.mahovik/upgrade-openssl-for-macos-e7a9ed82a76b "
+	exit
+fi
```

Comment:
Aren't these checks and msgs only appropriate on macos? It should be surrounded by `if [[ $OSTYPE == darwin* ]]; then`